spec/pluralrules.html

```diff
@@ -151,23 +141,13 @@
         1. Assert: Type(_n_) is Number.
         1. If isFinite(_n_) is *false*, then
           1. Return *"other"*.
-        1. If the _pluralRules_.[[MinimumSignificantDigits]] is not *undefined* and _pluralRules_.[[MaximumSignificantDigits]] is not *undefined*, then
-          1. Let _n_ be ToRawPrecision(_n_, _pluralRules_.[[MinimumSignificantDigits]], _pluralRules_.[[MaximumSignificantDigits]]).
-        1. Else,
-          1. Let _n_ be ToRawFixed(_n_, _pluralRules_.[[MinimumIntegerDigits]], _pluralRules_.[[MinimumFractionDigits]], _pluralRules_.[[MaximumFractionDigits]]).
+        1. Let _s_ be the result of calling the FormatNumber abstract operation with arguments _pluralRules_.[[NumberFormat]] and _n_.
```

Comment:
I was copying the style used in [ECMA-402](http://www.ecma-international.org/ecma-402/1.0/); if you search it for "FormatNumber", this is the expression that's used there four times. Should that not be the style to follow here?